backend/db/seeds.rb

```diff
@@ -279,7 +279,7 @@
   enrolled: 0,
   course: course,
   created_by_id: creator.id,
-  tutor_id: creator.id
+  tutor_id: creator.id,
```

Comment:
ya que tiene conflictos cuando los arreglen de paso saquen la coma juju
```suggestion
  tutor_id: creator.id
```